systemvm/patches/debian/config/opt/cloud/bin/update_config.py

```diff
@@ -52,15 +52,16 @@ def process(do_merge=True):
     qf.setFile(sys.argv[1])
     qf.do_merge = do_merge
     qf.load(None)
-
     return qf
 
 
 def process_file():
     print "[INFO] process_file"
     qf = process()
-    # Converge
-    finish_config()
+    # These can be safely deferred, dramatically speeding up loading times
+    if not (os.environ.get('DEFER_CONFIG', False) and sys.argv[1] in ('vm_dhcp_entry.json', 'vm_metadata.json')):
```

Comment:
in case of dhcp or metadata from vr_cfg.sh, we will leave the finish_config in /opt/cloud/bin/configure.py vm_dhcp_entry.json
or 
/opt/cloud/bin/configure.py vm_metadata.json

for others, finish_config should be executed.